src/main/java/com/github/manolo8/darkbot/utils/LibUtils.java

```diff
@@ -0,0 +1,13 @@
+package com.github.manolo8.darkbot.utils;
+
+import java.nio.file.Paths;
+
+public class LibUtils {
+    /**
+     * Loads specified library with improved precision of path, goals to avoid loading from unwanted places.
+     * @param library Filename of library including extension, e.g. "DarkBoatAPI.dll"
+     */
+    public static void loadLibrary(String library) {
+        System.load(Paths.get("lib", library).toAbsolutePath().toString());
+    }
```

Comment:
Don't call system.load from here, make this util just about returning a path, and call system.load in the class where it is used:

`System.load(LibUtils.getPath("DarkBoatAPI"));`
Also remove the need for the extension when calling, and add in the dll or so depending on the OS it runs on